examples/bri-3/prisma/schema.prisma

```diff
@@ -16,13 +16,22 @@ model BpiSubject {
   description String
   type        Int
   publicKey   String
+  OwnedBpiSubjectAccounts BpiSubjectAccount[] @relation(name: "ownerBpiSubject_fk")
+  CreatedBpiSubjectAccounts BpiSubjectAccount[] @relation(name: "creatorBpiSubject_fk")
 }
 
 model BpiAccount {
   id    String @id @default(uuid())
   nonce Int
 }
 
+model BpiSubjectAccount {
```

Comment:
@skosito hate to be a nagger ... when you look at [R80](https://github.com/eea-oasis/baseline-standard/blob/main/core/baseline-core-v1.0-psd01.md#r80) you see that several properties are missing. Please, add if only as placeholders. And then please cascade the changes down to handlers, class, etc.

cc @ognjenkurtic @Kasshern @Manik-Jain @Ybittan

Reply:
copy of my comment from PR #541 

@Therecanbeonlyone1969 yes, let's please stick with the proposed models from the document @skosito outlined. As soon as CRUD milestone is reached we will start tackling all the intricate details of state storage and other MUST requirements outlined in the standard. These will be tackled in separate smaller issues to avoid tackling multiple complex topics at the same time and to enable detailed discussions in the issues before implementation even begins. Dont worry,. no MUST requirement outlined in the scope document will be left behind ;).

Reply:
Ok as discussed